ffi/dotnet/Devolutions.IronRdp/Devolutions.IronRdp.Build.props

```diff
@@ -0,0 +1,108 @@
+<Project>
+  <PropertyGroup>
+    <TargetFrameworks>net8.0;Xamarin.iOS10</TargetFrameworks>
```

Comment:
I would drop the Xamarin target at this point

Reply:
Sounds good!